.github/workflows/publish_intellij_plugin.yml

```diff
@@ -0,0 +1,43 @@
+name: Build and Publish Intellij Plugin
+
+on:
+  push:
+    branches:
+      - prod-sync
+    paths:
+      - '_data/latest/**'
+    
+jobs:
+  trigger_workflow:
+     strategy:
+       matrix:
+         os: [ubuntu-latest]
+     runs-on: ${{ matrix.os }}
```

Comment:
We can directly set the runner as below right. No need a matrix.

https://github.com/ballerina-platform/ballerina-dev-website/blob/master/.github/workflows/add_new_release_note.yml#L12